hydromt_fiat/data_apis/open_street_maps.py

```diff
@@ -38,10 +38,17 @@ def get_roads_from_osm(
         "highway": road_types
     }  # this is the tag we use to find the correct OSM data
     
-    roads = ox.features.features_from_polygon(
-        polygon, tags=tag
-    )  # then we query the data
-
+    # Make sure that polygon is valid
+    if not polygon.is_valid:
```

Comment:
I think the validity has to do with the Polygon and if it is closed etc. This can be an issue with rounding errors. At least I had this issue when using a watershed polygon from a sfincs simulation and this solved it.

Reply:
ah okay!